sdk/src/main/assets/hcaptcha-form.html

```diff
@@ -101,13 +101,12 @@
             callback: function callback(token) {
                 return BridgeObject.onPass(token);
             },
-            'expired-callback': function expiredCallback() {
-                return BridgeObject.onError(15);
-            },
             'chalexpired-callback': function chalexpiredCallback() {
+                BridgeObject.onChallengeExpired();
                 return BridgeObject.onError(15);
             },
             'close-callback': function closeCallback() {
+                BridgeObject.onClose();
```

Comment:
This is redundant and can be removed.

Reply:
Agree about `BridgeObject.onClose();` but `BridgeObject.onError(30)` probably should stay because 

```
...
    document.body.onclick = function () {
        if (!window.hcaptcha) {
            BridgeObject.onError(30);
        }
    };
```

called only when `!window.hcaptcha`, am I missing something?

Reply:
Yes, `BridgeObject.onError(30)` should stay.